lms/templates/registration/password_reset_complete.html

```diff
@@ -1,35 +1,39 @@
-{% extends "main_django.html" %}
-{% load i18n %}
+## mako
 
-{% block title %}
-<title>{% trans "Your Password Reset is Complete" %}</title>
-{% endblock %}
+<%! from django.utils.translation import ugettext as _ %>
 
-{% block bodyextra %}
+<%inherit file="../main.html"/>
+
+<%block name="title">
+    <title>${_("Your Password Reset is Complete")}</title>
+</%block>
+
+<%block name="bodyextra">
     <script type="text/javascript">
         $(function() {
             'use strict';
             // adding js class for styling with accessibility in mind
             $('body').addClass('js');
         });
     </script>
-{% endblock %}
+</%block>
 
-{% block bodyclass %}view-passwordreset{% endblock %}
+<%block name="bodyclass">view-passwordreset</%block>
 
-{% block body %}
+<%block name="body">
 <div id="password-reset-complete-container" class="login-register">
     <section id="password-reset-complete-anchor" class="form-type">
         <div id="password-reset-complete" class="form-wrapper">
             <div class="status submission-success" aria-live="polite">
-                <h4 class="message-title">{% trans "Password Reset Complete" %}</h4>
+                <h4 class="message-title">${_("Password Reset Complete")}</h4>
                 <ul class="message-copy">
-                {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
-                    Your password has been reset. {{ link_start }}Sign in to your account.{{ link_end }}
-                {% endblocktrans %}
+                ${_(
```

Comment:
Follow the same safe templating approach here: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html